2025-06-05 09:22:19,101 - INFO - Aplicación interrumpida por el usuario (Ctrl+C).
2025-06-05 09:22:19,101 - INFO - Cerrando aplicación...
2025-06-05 09:22:19,105 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 09:22:19,108 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 09:22:19,130 - INFO - Script de aplicación de backup finalizado.
2025-06-05 09:22:23,583 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 09:22:26,276 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 09:22:26,278 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 09:22:26,282 - INFO - Iniciando intento de lógica de backup...
2025-06-05 09:22:26,285 - INFO - Archivo 'backup_info.txt' no encontrado, iniciando contador de backup en 1.
2025-06-05 09:22:26,287 - INFO - Ejecutando: git status --porcelain
2025-06-05 09:22:26,351 - INFO - Verificar Estado Git finalizado. Código: 0
Salida Estándar:
M backup_git.log
 D backup_info.txt

2025-06-05 09:22:26,352 - INFO - Cambios detectados o archivos no rastreados:
M backup_git.log
 D backup_info.txt
2025-06-05 09:22:26,354 - INFO - Ejecutando: git add .
2025-06-05 09:22:26,430 - INFO - Git Add finalizado. Código: 0

2025-06-05 09:22:26,431 - INFO - Realizando commit con mensaje: 'Backup automatico - 2025-06-05 09:22:26'...
2025-06-05 09:22:26,435 - INFO - Ejecutando: git commit -m Backup automatico - 2025-06-05 09:22:26
2025-06-05 09:22:26,552 - INFO - Git Commit finalizado. Código: 0
Salida Estándar:
[master c384f3c] Backup automatico - 2025-06-05 09:22:26
 2 files changed, 5 insertions(+), 786 deletions(-)
 delete mode 100644 backup_info.txt

2025-06-05 09:22:26,553 - INFO - Commit realizado exitosamente.
2025-06-05 09:22:26,558 - INFO - Realizando push al repositorio remoto (git push)...
2025-06-05 09:22:26,564 - INFO - Ejecutando: git push
2025-06-05 09:22:28,395 - INFO - Git Push finalizado. Código: 0
Salida de Error:
To https://github.com/josdeath/frontendChat.git
   02eff85..c384f3c  master -> master

2025-06-05 09:22:28,396 - INFO - Push realizado exitosamente.
2025-06-05 09:22:28,409 - INFO - Número de backup 1 guardado en 'backup_info.txt'.
2025-06-05 09:22:28,420 - INFO - Intento 1 finalizado en 2.14s con estado: SUCCESS
2025-06-05 09:22:28,432 - INFO - Backup completado exitosamente después de 1 intento(s).
2025-06-05 09:22:28,439 - INFO - Duración total del proceso: 2.15 segundos.
2025-06-05 09:24:33,179 - INFO - Aplicación interrumpida por el usuario (Ctrl+C).
2025-06-05 09:24:33,179 - INFO - Cerrando aplicación...
2025-06-05 09:24:33,183 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 09:24:33,187 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 09:24:33,205 - INFO - Script de aplicación de backup finalizado.
2025-06-05 09:28:25,328 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 09:28:27,734 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 09:28:27,736 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 09:28:27,740 - INFO - Iniciando intento de lógica de backup...
2025-06-05 09:28:27,744 - INFO - Último número de backup leído: 1. Siguiente número: 2
2025-06-05 09:28:27,745 - INFO - Ejecutando: git status --porcelain
2025-06-05 09:28:27,815 - INFO - Verificar Estado Git finalizado. Código: 0
Salida Estándar:
M backup_git.log
 M intro.py
?? backup_info.txt

2025-06-05 09:28:27,815 - INFO - Cambios detectados o archivos no rastreados:
M backup_git.log
 M intro.py
?? backup_info.txt
2025-06-05 09:28:27,818 - INFO - Ejecutando: git add .
2025-06-05 09:28:27,900 - INFO - Git Add finalizado. Código: 0

2025-06-05 09:28:27,901 - INFO - Realizando commit con mensaje: 'Backup #2 - 2025-06-05 09:28:27'...
2025-06-05 09:28:27,905 - INFO - Ejecutando: git commit -m Backup #2 - 2025-06-05 09:28:27
2025-06-05 09:28:28,115 - INFO - Git Commit finalizado. Código: 0
Salida Estándar:
[master e4e992a] Backup #2 - 2025-06-05 09:28:27
 3 files changed, 45 insertions(+), 1 deletion(-)
 create mode 100644 backup_info.txt

2025-06-05 09:28:28,115 - INFO - Commit realizado exitosamente.
2025-06-05 09:28:28,121 - INFO - Realizando push al repositorio remoto (git push)...
2025-06-05 09:28:28,129 - INFO - Ejecutando: git push
2025-06-05 09:28:30,181 - INFO - Git Push finalizado. Código: 0
Salida de Error:
To https://github.com/josdeath/frontendChat.git
   c384f3c..e4e992a  master -> master

2025-06-05 09:28:30,182 - INFO - Push realizado exitosamente.
2025-06-05 09:28:30,191 - INFO - Número de backup 2 guardado en 'backup_info.txt'.
2025-06-05 09:28:30,201 - INFO - Intento 1 finalizado en 2.46s con estado: SUCCESS
2025-06-05 09:28:30,208 - INFO - Backup completado exitosamente después de 1 intento(s).
2025-06-05 09:28:30,221 - INFO - Duración total del proceso: 2.47 segundos.
2025-06-05 09:28:34,386 - INFO - Cerrando aplicación...
2025-06-05 09:28:34,389 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 09:28:34,392 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 09:28:34,425 - INFO - Aplicación de backup cerrada por el usuario (mainloop terminado).
2025-06-05 09:28:34,425 - INFO - Script de aplicación de backup finalizado.
2025-06-05 09:35:57,421 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 09:36:55,820 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 09:36:55,821 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 09:36:55,823 - INFO - Iniciando intento de lógica de backup...
2025-06-05 09:36:55,826 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 09:36:55,827 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 09:36:55,844 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 09:36:55,845 - INFO - Intento 1 finalizado en 0.02s con estado: CONNECTION_ERROR
2025-06-05 09:36:55,848 - ERROR - Intento 1 falló. Estado: CONNECTION_ERROR
2025-06-05 09:36:55,853 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 09:37:05,857 - INFO - --- Iniciando Intento de Backup 2/3 ---
2025-06-05 09:37:05,859 - INFO - Iniciando intento de lógica de backup...
2025-06-05 09:37:05,867 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 09:37:05,877 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 09:37:05,887 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 09:37:05,897 - INFO - Intento 2 finalizado en 0.04s con estado: CONNECTION_ERROR
2025-06-05 09:37:05,902 - ERROR - Intento 2 falló. Estado: CONNECTION_ERROR
2025-06-05 09:37:05,907 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 09:37:15,914 - INFO - --- Iniciando Intento de Backup 3/3 ---
2025-06-05 09:37:15,916 - INFO - Iniciando intento de lógica de backup...
2025-06-05 09:37:15,927 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 09:37:15,933 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 09:37:15,939 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 09:37:15,946 - INFO - Intento 3 finalizado en 0.03s con estado: CONNECTION_ERROR
2025-06-05 09:37:15,954 - ERROR - Intento 3 falló. Estado: CONNECTION_ERROR
2025-06-05 09:37:15,970 - ERROR - Todos los 3 intentos de backup fallaron.
2025-06-05 09:37:15,981 - ERROR - Duración total del proceso (fallido): 20.15 segundos.
2025-06-05 09:48:01,852 - INFO - Cerrando aplicación...
2025-06-05 09:48:01,860 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 09:48:01,862 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 09:48:01,915 - INFO - Aplicación de backup cerrada por el usuario (mainloop terminado).
2025-06-05 09:48:01,916 - INFO - Script de aplicación de backup finalizado.
2025-06-05 09:48:04,557 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 09:48:07,053 - INFO - Cerrando aplicación...
2025-06-05 09:48:07,054 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 09:48:07,055 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 09:48:07,099 - INFO - Aplicación de backup cerrada por el usuario (mainloop terminado).
2025-06-05 09:48:07,099 - INFO - Script de aplicación de backup finalizado.
2025-06-05 09:48:15,264 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 09:48:36,756 - INFO - Aplicación interrumpida por el usuario (Ctrl+C).
2025-06-05 09:48:36,757 - INFO - Cerrando aplicación...
2025-06-05 09:48:36,759 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 09:48:36,761 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 09:48:36,831 - INFO - Script de aplicación de backup finalizado.
2025-06-05 11:42:18,221 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 11:42:22,054 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 11:42:22,056 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 11:42:22,059 - INFO - Iniciando intento de lógica de backup...
2025-06-05 11:42:22,063 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 11:42:22,065 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 11:42:22,138 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 11:42:22,139 - INFO - Intento 1 finalizado en 0.08s con estado: CONNECTION_ERROR
2025-06-05 11:42:22,142 - ERROR - Intento 1 falló. Estado: CONNECTION_ERROR
2025-06-05 11:42:22,144 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 11:42:32,151 - INFO - --- Iniciando Intento de Backup 2/3 ---
2025-06-05 11:42:32,153 - INFO - Iniciando intento de lógica de backup...
2025-06-05 11:42:32,161 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 11:42:32,172 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 11:42:32,182 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 11:42:32,189 - INFO - Intento 2 finalizado en 0.04s con estado: CONNECTION_ERROR
2025-06-05 11:42:32,205 - ERROR - Intento 2 falló. Estado: CONNECTION_ERROR
2025-06-05 11:42:32,211 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 11:42:42,220 - INFO - --- Iniciando Intento de Backup 3/3 ---
2025-06-05 11:42:42,221 - INFO - Iniciando intento de lógica de backup...
2025-06-05 11:42:42,227 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 11:42:42,233 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 11:42:42,244 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 11:42:42,257 - INFO - Intento 3 finalizado en 0.04s con estado: CONNECTION_ERROR
2025-06-05 11:42:42,271 - ERROR - Intento 3 falló. Estado: CONNECTION_ERROR
2025-06-05 11:42:42,282 - ERROR - Todos los 3 intentos de backup fallaron.
2025-06-05 11:42:42,295 - ERROR - Duración total del proceso (fallido): 20.23 segundos.
2025-06-05 11:43:01,593 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 11:43:01,594 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 11:43:01,595 - INFO - Iniciando intento de lógica de backup...
2025-06-05 11:43:01,598 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 11:43:01,603 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 11:43:01,605 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 11:43:01,606 - INFO - Intento 1 finalizado en 0.01s con estado: CONNECTION_ERROR
2025-06-05 11:43:01,610 - ERROR - Intento 1 falló. Estado: CONNECTION_ERROR
2025-06-05 11:43:01,616 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 11:43:11,621 - INFO - --- Iniciando Intento de Backup 2/3 ---
2025-06-05 11:43:11,623 - INFO - Iniciando intento de lógica de backup...
2025-06-05 11:43:11,627 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 11:43:11,635 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 11:43:11,640 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 11:43:11,645 - INFO - Intento 2 finalizado en 0.02s con estado: CONNECTION_ERROR
2025-06-05 11:43:11,647 - ERROR - Intento 2 falló. Estado: CONNECTION_ERROR
2025-06-05 11:43:11,653 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 11:43:21,659 - INFO - --- Iniciando Intento de Backup 3/3 ---
2025-06-05 11:43:21,660 - INFO - Iniciando intento de lógica de backup...
2025-06-05 11:43:21,679 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 11:43:21,687 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 11:43:21,690 - ERROR - No se pudo conectar a GitHub: [Errno 11001] getaddrinfo failed
2025-06-05 11:43:21,692 - INFO - Intento 3 finalizado en 0.03s con estado: CONNECTION_ERROR
2025-06-05 11:43:21,698 - ERROR - Intento 3 falló. Estado: CONNECTION_ERROR
2025-06-05 11:43:21,704 - ERROR - Todos los 3 intentos de backup fallaron.
2025-06-05 11:43:21,706 - ERROR - Duración total del proceso (fallido): 20.11 segundos.
2025-06-05 11:43:26,034 - INFO - Cerrando aplicación...
2025-06-05 11:43:26,039 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 11:43:26,044 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 11:43:26,089 - INFO - Aplicación de backup cerrada por el usuario (mainloop terminado).
2025-06-05 11:43:26,090 - INFO - Script de aplicación de backup finalizado.
2025-06-05 11:57:16,948 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 11:57:22,451 - INFO - Cerrando aplicación...
2025-06-05 11:57:22,452 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 11:57:22,452 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 11:57:22,492 - INFO - Aplicación de backup cerrada por el usuario (mainloop terminado).
2025-06-05 11:57:22,492 - INFO - Script de aplicación de backup finalizado.
2025-06-05 11:57:42,397 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 11:57:43,656 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 11:57:43,658 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 11:57:43,664 - INFO - Iniciando intento de lógica de backup...
2025-06-05 11:57:43,668 - INFO - Último número de backup leído: 2. Siguiente número: 3
2025-06-05 11:57:43,671 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 11:57:43,775 - INFO - Conexión a GitHub establecida correctamente.
2025-06-05 11:57:43,776 - INFO - Ejecutando: git status --porcelain
2025-06-05 11:57:43,913 - INFO - Verificar Estado Git finalizado. Código: 0
Salida Estándar:
M .gitignore
 M backup_git.log
 M backup_info.txt
 D intro.py
?? backupGit.py

2025-06-05 11:57:43,914 - INFO - Cambios detectados o archivos no rastreados:
M .gitignore
 M backup_git.log
 M backup_info.txt
 D intro.py
?? backupGit.py
2025-06-05 11:57:43,920 - INFO - Ejecutando: git add .
2025-06-05 11:57:44,022 - INFO - Git Add finalizado. Código: 0
Salida de Error:
warning: in the working copy of '.gitignore', LF will be replaced by CRLF the next time Git touches it

2025-06-05 11:57:44,023 - INFO - Realizando commit con mensaje: 'Backup #3 - 2025-06-05 11:57:44'...
2025-06-05 11:57:44,026 - INFO - Ejecutando: git commit -m Backup #3 - 2025-06-05 11:57:44
2025-06-05 11:57:44,211 - INFO - Git Commit finalizado. Código: 0
Salida Estándar:
[master 1c8e763] Backup #3 - 2025-06-05 11:57:44
 4 files changed, 200 insertions(+), 3 deletions(-)
 rename intro.py => backupGit.py (93%)

2025-06-05 11:57:44,212 - INFO - Commit realizado exitosamente.
2025-06-05 11:57:44,222 - INFO - Realizando push al repositorio remoto (git push)...
2025-06-05 11:57:44,232 - INFO - Ejecutando: git push
2025-06-05 11:57:46,552 - INFO - Git Push finalizado. Código: 0
Salida de Error:
To https://github.com/josdeath/frontendChat.git
   e4e992a..1c8e763  master -> master

2025-06-05 11:57:46,554 - INFO - Push realizado exitosamente.
2025-06-05 11:57:46,565 - INFO - Número de backup 3 guardado en 'backup_info.txt'.
2025-06-05 11:57:46,590 - INFO - Intento 1 finalizado en 2.93s con estado: SUCCESS
2025-06-05 11:57:46,604 - INFO - Backup completado exitosamente después de 1 intento(s).
2025-06-05 11:57:46,613 - INFO - Duración total del proceso: 2.95 segundos.
2025-06-05 11:58:07,705 - INFO - Cerrando aplicación...
2025-06-05 11:58:07,713 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 11:58:07,717 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 11:58:07,767 - INFO - Aplicación de backup cerrada por el usuario (mainloop terminado).
2025-06-05 11:58:07,767 - INFO - Script de aplicación de backup finalizado.
2025-06-05 12:04:08,394 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 12:04:09,761 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 12:04:09,762 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 12:04:09,763 - INFO - Iniciando intento de lógica de backup...
2025-06-05 12:04:09,764 - INFO - Último número de backup leído: 3. Siguiente número: 4
2025-06-05 12:04:09,766 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 12:04:09,866 - INFO - Conexión a GitHub establecida correctamente.
2025-06-05 12:04:09,866 - INFO - Ejecutando: git status --porcelain
2025-06-05 12:04:10,010 - INFO - Verificar Estado Git finalizado. Código: 0
Salida Estándar:
M .gitignore
 M backupGit.py
 M backup_git.log
 M backup_info.txt

2025-06-05 12:04:10,012 - INFO - Cambios detectados o archivos no rastreados:
M .gitignore
 M backupGit.py
 M backup_git.log
 M backup_info.txt
2025-06-05 12:04:10,014 - INFO - Ejecutando: git add .
2025-06-05 12:04:10,165 - INFO - Git Add finalizado. Código: 0
Salida de Error:
warning: in the working copy of '.gitignore', LF will be replaced by CRLF the next time Git touches it

2025-06-05 12:04:10,166 - INFO - Realizando commit con mensaje: 'Backup #4 - 2025-06-05 12:04:10'...
2025-06-05 12:04:10,168 - INFO - Ejecutando: git commit -m Backup #4 - 2025-06-05 12:04:10
2025-06-05 12:04:10,324 - INFO - Git Commit finalizado. Código: 0
Salida Estándar:
[master cb8490f] Backup #4 - 2025-06-05 12:04:10
 4 files changed, 55 insertions(+), 7 deletions(-)

2025-06-05 12:04:10,324 - INFO - Commit realizado exitosamente.
2025-06-05 12:04:10,332 - INFO - Realizando push al repositorio remoto (git push)...
2025-06-05 12:04:10,348 - INFO - Ejecutando: git push
2025-06-05 12:04:10,777 - ERROR - Intento 1 excedió el tiempo límite de 0 minutos.
2025-06-05 12:04:10,777 - INFO - Intento 1 finalizado en 1.01s con estado: TIMEOUT_ATTEMPT
2025-06-05 12:04:10,784 - ERROR - Intento 1 falló. Estado: TIMEOUT_ATTEMPT
2025-06-05 12:04:10,787 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 12:04:12,750 - INFO - Git Push finalizado. Código: 0
Salida de Error:
To https://github.com/josdeath/frontendChat.git
   1c8e763..cb8490f  master -> master

2025-06-05 12:04:12,752 - INFO - Push realizado exitosamente.
2025-06-05 12:04:12,767 - INFO - Número de backup 4 guardado en 'backup_info.txt'.
2025-06-05 12:04:20,799 - INFO - --- Iniciando Intento de Backup 2/3 ---
2025-06-05 12:04:20,800 - INFO - Iniciando intento de lógica de backup...
2025-06-05 12:04:20,808 - INFO - Último número de backup leído: 4. Siguiente número: 5
2025-06-05 12:04:20,815 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 12:04:20,862 - INFO - Conexión a GitHub establecida correctamente.
2025-06-05 12:04:20,863 - INFO - Ejecutando: git status --porcelain
2025-06-05 12:04:20,951 - INFO - Verificar Estado Git finalizado. Código: 0
Salida Estándar:
M backup_git.log
 M backup_info.txt

2025-06-05 12:04:20,952 - INFO - Cambios detectados o archivos no rastreados:
M backup_git.log
 M backup_info.txt
2025-06-05 12:04:20,959 - INFO - Ejecutando: git add .
2025-06-05 12:04:21,032 - INFO - Git Add finalizado. Código: 0

2025-06-05 12:04:21,033 - INFO - Realizando commit con mensaje: 'Backup #5 - 2025-06-05 12:04:21'...
2025-06-05 12:04:21,038 - INFO - Ejecutando: git commit -m Backup #5 - 2025-06-05 12:04:21
2025-06-05 12:04:21,184 - INFO - Git Commit finalizado. Código: 0
Salida Estándar:
[master b48d3ec] Backup #5 - 2025-06-05 12:04:21
 2 files changed, 40 insertions(+), 1 deletion(-)

2025-06-05 12:04:21,185 - INFO - Commit realizado exitosamente.
2025-06-05 12:04:21,189 - INFO - Realizando push al repositorio remoto (git push)...
2025-06-05 12:04:21,207 - INFO - Ejecutando: git push
2025-06-05 12:04:21,801 - ERROR - Intento 2 excedió el tiempo límite de 0 minutos.
2025-06-05 12:04:21,802 - INFO - Intento 2 finalizado en 1.00s con estado: TIMEOUT_ATTEMPT
2025-06-05 12:04:21,804 - ERROR - Intento 2 falló. Estado: TIMEOUT_ATTEMPT
2025-06-05 12:04:21,817 - INFO - Esperando 10s antes del próximo intento...
2025-06-05 12:04:23,340 - INFO - Git Push finalizado. Código: 0
Salida de Error:
To https://github.com/josdeath/frontendChat.git
   cb8490f..b48d3ec  master -> master

2025-06-05 12:04:23,341 - INFO - Push realizado exitosamente.
2025-06-05 12:04:23,347 - INFO - Número de backup 5 guardado en 'backup_info.txt'.
2025-06-05 12:04:31,829 - INFO - --- Iniciando Intento de Backup 3/3 ---
2025-06-05 12:04:31,830 - INFO - Iniciando intento de lógica de backup...
2025-06-05 12:04:31,838 - INFO - Último número de backup leído: 5. Siguiente número: 6
2025-06-05 12:04:31,852 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 12:04:31,914 - INFO - Conexión a GitHub establecida correctamente.
2025-06-05 12:04:31,915 - INFO - Ejecutando: git status --porcelain
2025-06-05 12:04:32,017 - INFO - Verificar Estado Git finalizado. Código: 0
Salida Estándar:
M backup_git.log
 M backup_info.txt

2025-06-05 12:04:32,017 - INFO - Cambios detectados o archivos no rastreados:
M backup_git.log
 M backup_info.txt
2025-06-05 12:04:32,024 - INFO - Ejecutando: git add .
2025-06-05 12:04:32,099 - INFO - Git Add finalizado. Código: 0

2025-06-05 12:04:32,099 - INFO - Realizando commit con mensaje: 'Backup #6 - 2025-06-05 12:04:32'...
2025-06-05 12:04:32,107 - INFO - Ejecutando: git commit -m Backup #6 - 2025-06-05 12:04:32
2025-06-05 12:04:32,253 - INFO - Git Commit finalizado. Código: 0
Salida Estándar:
[master 7efc28f] Backup #6 - 2025-06-05 12:04:32
 2 files changed, 39 insertions(+), 1 deletion(-)

2025-06-05 12:04:32,254 - INFO - Commit realizado exitosamente.
2025-06-05 12:04:32,260 - INFO - Realizando push al repositorio remoto (git push)...
2025-06-05 12:04:32,267 - INFO - Ejecutando: git push
2025-06-05 12:04:32,844 - ERROR - Intento 3 excedió el tiempo límite de 0 minutos.
2025-06-05 12:04:32,845 - INFO - Intento 3 finalizado en 1.01s con estado: TIMEOUT_ATTEMPT
2025-06-05 12:04:32,854 - ERROR - Intento 3 falló. Estado: TIMEOUT_ATTEMPT
2025-06-05 12:04:32,866 - ERROR - Todos los 3 intentos de backup fallaron.
2025-06-05 12:04:32,869 - ERROR - Duración total del proceso (fallido): 23.10 segundos.
2025-06-05 12:04:34,331 - INFO - Git Push finalizado. Código: 0
Salida de Error:
To https://github.com/josdeath/frontendChat.git
   b48d3ec..7efc28f  master -> master

2025-06-05 12:04:34,332 - INFO - Push realizado exitosamente.
2025-06-05 12:04:34,338 - INFO - Número de backup 6 guardado en 'backup_info.txt'.
2025-06-05 12:07:54,445 - INFO - Aplicación interrumpida por el usuario (Ctrl+C).
2025-06-05 12:07:54,445 - INFO - Cerrando aplicación...
2025-06-05 12:07:54,449 - INFO - Intentando cerrar el ThreadPoolExecutor...
2025-06-05 12:07:54,452 - INFO - ThreadPoolExecutor cerrado.
2025-06-05 12:07:54,467 - INFO - Script de aplicación de backup finalizado.
2025-06-05 12:07:56,178 - INFO - Aplicación de backup iniciada en un repositorio Git.
2025-06-05 12:07:58,369 - INFO - Iniciando proceso de backup con reintentos...
2025-06-05 12:07:58,370 - INFO - --- Iniciando Intento de Backup 1/3 ---
2025-06-05 12:07:58,374 - INFO - Iniciando intento de lógica de backup...
2025-06-05 12:07:58,378 - INFO - Último número de backup leído: 6. Siguiente número: 7
2025-06-05 12:07:58,379 - INFO - Verificando conexión a GitHub (github.com:443)...
2025-06-05 12:07:58,462 - INFO - Conexión a GitHub establecida correctamente.
2025-06-05 12:07:58,464 - INFO - Ejecutando: git status --porcelain
2025-06-05 12:07:58,532 - INFO - Verificar Estado Git finalizado. Código: 0
Salida Estándar:
M backupGit.py
 M backup_git.log
 M backup_info.txt

2025-06-05 12:07:58,533 - INFO - Cambios detectados o archivos no rastreados:
M backupGit.py
 M backup_git.log
 M backup_info.txt
2025-06-05 12:07:58,537 - INFO - Ejecutando: git add .
2025-06-05 12:07:58,625 - INFO - Git Add finalizado. Código: 0

2025-06-05 12:07:58,626 - INFO - Realizando commit con mensaje: 'Backup #7 - 2025-06-05 12:07:58'...
2025-06-05 12:07:58,633 - INFO - Ejecutando: git commit -m Backup #7 - 2025-06-05 12:07:58
2025-06-05 12:07:58,810 - INFO - Git Commit finalizado. Código: 0
Salida Estándar:
[master 5084e56] Backup #7 - 2025-06-05 12:07:58
 3 files changed, 52 insertions(+), 2 deletions(-)

2025-06-05 12:07:58,810 - INFO - Commit realizado exitosamente.
2025-06-05 12:07:58,818 - INFO - Realizando push al repositorio remoto (git push)...
2025-06-05 12:07:58,821 - INFO - Ejecutando: git push
2025-06-05 12:07:59,384 - ERROR - Intento 1 excedió el tiempo límite de 0 minutos.
2025-06-05 12:07:59,385 - INFO - Intento 1 finalizado en 1.01s con estado: TIMEOUT_ATTEMPT
2025-06-05 12:07:59,398 - ERROR - Intento 1 falló. Estado: TIMEOUT_ATTEMPT
2025-06-05 12:07:59,411 - INFO - Esperando 10s antes del próximo intento...
